src/main/java/nl/tudelft/cse1110/andy/config/RunConfiguration.java

```diff
@@ -61,4 +61,12 @@ public String successMessage() {
         return null;
     }
 
+    public boolean skipJacoco() {
```

Comment:
do we need this?

why not, in the `getRunConfiguration().skipJacoco()`, we simply do ``return weight == 0``?

Reply:
I would say it is nice to have the flexibility to disable it only when strictly necessary due to them breaking things (for instance, in Selenium tests, where there's no library class). I think those results should be shown to students even in cases where they are not taken into account for the final score. For instance, on an exercise whose final score is solely determined by meta tests and code checks, it may well still be useful for students to see code coverage information.